deploy-templates/jellyfish-product/product/config-manifest.yml

```diff
@@ -6,8 +6,6 @@ properties:
     type: string
   JELLYFISH_API_HOST:
     type: string
-  JELLYFISH_LIVECHAT_HOST:
-    type: string
```

Comment:
Rebased after merging https://github.com/product-os/jellyfish/pull/8131, but noticed `JELLYFISH_LIVECHAT_HOST` is being removed in this PR. The current livechat ingress (that still exists) depends on this variable. Are we OK with dropping `livechat.ly.fish` entirely or are there some other steps that need to take place beforehand so as to not interrupt support?

Reply:
@joshbwlng we need to drop livechat entirely, we only need domain name to direct to ui service. Support chat already works with jel.ly.fish/livechat embedded in iframe. If I delete this configuration, will it prevent livechat.ly.fish redirection?

Reply:
@karaxuna We can drop all k8s configurations if we no longer need to provide anything at `livechat.ly.fish`. The current directing of requests from `livechat.ly.fish` to ui pods is because of the livechat ingress config and this variable. So, deleting the livechat ingress and/or this variable will prevent redirection. Does support chat not use the `livechat.ly.fish` domain?

Reply:
@joshbwlng we should use livechat.ly.fish domain as iframe src in order to have different sessions on dashboard and on jellyfish. Currently both work with jel.ly.fish domain that causes shared localStorage. I'll leave k8s config untouched and just remove everything else.